src/DssSpell.t.sol

```diff
@@ -2494,15 +2252,53 @@ contract DssSpellTest is DSTest, DSMath {
         // Hacking nxt price to 0x123 (and making it valid)
         bytes32 hackedValue = 0x0000000000000000000000000000000100000000000000000000000000000123;
 
-        ClipAbstract clipLINKA = ClipAbstract(addr.addr("MCD_CLIP_LINK_A"));
+        ClipAbstract clipYFIA = ClipAbstract(addr.addr("MCD_CLIP_YFI_A"));
 
-        hevm.store(address(addr.addr("PIP_LINK")), bytes32(uint256(4)), hackedValue);
+        hevm.store(address(addr.addr("PIP_YFI")), bytes32(uint256(4)), hackedValue);
 
-        assertEq(clipMom.tolerance(address(clipLINKA)), (RAY / 2)); // (RAY / 2) for 50%
+        assertEq(clipMom.tolerance(address(clipYFIA)), (RAY / 2)); // (RAY / 2) for 50%
 
         // Price is hacked, anyone can trip the breaker
-        clipMom.tripBreaker(address(clipLINKA));
+        clipMom.tripBreaker(address(clipYFIA));
+
+        assertEq(clipYFIA.stopped(), 2);
+    }
+
+    function testLerp() public {
+        LerpFabLike factory = LerpFabLike(addr.addr("LERP_FAB"));
+
+        assertEq(vow.hump(), 30 * MILLION * RAD);
+        assertEq(factory.count(), 0);
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        assertEq(factory.count(), 1);
+        LerpLike lerp = LerpLike(factory.lerps("20210423_VOW_HUMP1"));
+
+        assertEq(vow.hump(), 30 * MILLION * RAD);
+
+        // Should do nothing as we are before the start date
+        lerp.tick();
+        assertEq(vow.hump(), 30 * MILLION * RAD);
+
```

Comment:
Missing `hevm.warp()` to the lerp `startTime`

Reply:
Yes this was removed due some issue making the test to pass but now that I'm looking at it in detail it shouldn't break anything. Adding it back.

Reply:
To give a context, we had to change a bit the structure of the tests as this spell requires the prev one being executed, but not like before that was only for comparing the risk values. In this case, as discussed, we needed it to allow the execution of the new one. That messed a bit the timestamp around the tests and we just didn't think about this case well enough.